app/Presenters/LicensePresenter.php

```diff
@@ -230,7 +230,17 @@ public static function dataTableLayoutSeats()
                 'title' => trans('admin/licenses/form.asset'),
                 'visible' => true,
                 'formatter' => 'hardwareLinkObjFormatter',
-            ], [
+            ],
+            [
+                "field" => "assigned_asset.serial",
+                "searchable" => false,
+                "sortable" => false,
+                "switchable" => true,
+                "title" => trans('admin/hardware/form.serial'),
```

Comment:
IIRC, this just says "serial", which could be confusing, since licenses have their own serials (product keys)